### ISSUE TITLE
Support for Jetson AGX Xavier

### ISSUE THREAD

**Issue by NewBermuda** [Picovoice/porcupine#459]
We were using Picovoice on Jetson Nano, which worked great. Now we moved to Jetson AGX Xavier and we thought it would be supported as well, since "Jetson" was supported.
However, we are getting the "platform" not supported error.

Could you please add support for Jetson AGX Xavier and tell us about a possible workaround until then?
Cheers

**Comment by kenarsa** [comment]
We don't have support for AGX, just Nano for now. Let's keep this open to measure traction and if it is significant we can add it to the roadmap.

**Comment by TomEelbode** [comment]
I'm wondering if support for Jetson AGX Orin platforms is part of the pipeline? I think that jetson nano is a bit outdated and that the newer product lines of NVIDIA Jetson are worth supporting.

**Comment by kenarsa** [comment]
@TomEelbode I agree, and we might discontinue nano support soon. I am unsure if we would add support for AGX without a leading commercial interest.
